conformance-test/globalHooks.ts

```diff
@@ -0,0 +1,35 @@
+/*!
+ * Copyright 2022 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  getTestBenchDockerImage,
+  runTestBenchDockerImage,
+  stopTestBenchDockerImage,
+} from './testBenchUtil';
+
+const TIMEOUT_FOR_DOCKER_OPS = 60000;
+
+export async function mochaGlobalSetup(this: any) {
```

Comment:
Disabled the explicit any for these functions as the `this` parameter had to be passed to avoid another linter issue, so either way was going to need to be disabled for this line / line below.